### ISSUE TITLE
package.json was removed from the example folder?

### ISSUE THREAD

**Issue by alvarotrigo** [alvarotrigo/react-fullpage#78]
Any reason why the package.json, README.md and so on was removed from the examples folder?
Was it to remove the additional webpack.config file?

How do we build the examples now? 

I noticed `webpack.config.js` file not being used in any npm script at the moment.
Should we perhaps add a npm script in the root package.json to build the example? And point it somewhere in the docs?

Ideally we can separate the examples dist files from the react-fullpage.js dist files?
Right now running it manually generate the `index.html` and `bundle.js` files in the `/dist` folder.

```sh
npx webpack --config webpack.config.js
``` 

Should we perhaps add an `output` object in the `webpack.config.js` file?

```js
  output: {
    filename: 'bundle.js',
    path: path.resolve(__dirname, './example/dist')
  },
```

**Comment by cmswalker** [comment]
It hasn't been doing anything for quite some time. Everything is controlled by the root level https://github.com/alvarotrigo/react-fullpage/blob/master/webpack.config.dev.js

Having nested configs is hard to follow, it never needed to be built to disk by using webpack-dev-server from root. `npm run dev` uses the root config but changes the directory to point to the example directory

**Comment by alvarotrigo** [comment]
When I run `npm run dev` the terminal never stops the task:
<img width="454" alt="screen shot 2018-11-22 at 19 00 45" src="https://user-images.githubusercontent.com/1706326/48917802-f947df00-ee88-11e8-8d10-c0fd03ca5800.png">

It generates `dist/index.html` (same dist folder as the react-fulllpage project) and I can't see any `bundle.js`.
Also, opening it will only show a black page with an error in the JS console:
<img width="550" alt="screen shot 2018-11-22 at 19 02 12" src="https://user-images.githubusercontent.com/1706326/48917847-2ac0aa80-ee89-11e8-88fb-565b1148aa51.png">

Please correct me if I'm doing something wrong :) Wouldn't be the 1st time!

**Comment by cmswalker** [comment]
to be safe I'd repull the repo and do a fresh `npm install` and then `npm run dev` from the root level. I just rebuild the repo from scratch and it seems ok

**Comment by cmswalker** [comment]
Just tested on both `master` and `dev` and it works.

Also not sure if it's obvious but `npm run dev` should never stop as its using `webpack-dev-server` not `npm run build` which writes to disk and exits

**Comment by alvarotrigo** [comment]
> Also not sure if it's obvious but `npm run dev` should never stop as its using `webpack-dev-server` not `npm run build` which writes to disk and exits

My mistake! Was forgetting the port number :)
Thanks for taking the time @cmswalker!

**Comment by alvarotrigo** [comment]
@cmswalker I've added a Readme.md in the example folder and a new script in package.json to generate the build files in the examples folder. 

Feel free to review if you want to change anything else.

**Comment by alvarotrigo** [comment]
I forgot to link the commit with the issue:
https://github.com/alvarotrigo/react-fullpage/commit/9b4f4dafeb4bb319487a9dea9e5fbb7226427201
